modules/op-constructing-pipelines-using-pipeline-builder.adoc

```diff
@@ -36,7 +36,7 @@ The *Pipeline builder* view supports a limited number of fields whereas the *YAM
 .YAML view
 image::op-pipeline-yaml.png[]
 +
-Configure your pipeline using the *Pipeline Builder*:
+Configure your pipeline using the *Pipeline builder*:
```

Comment:
Arguably, places like this should be un-capitalized - "using the pipeline builder". However, it is not consistent in other places in this same doc set (I could find both "the operator" and "the Operator"), so your call whether to change it. If you do, change in all places where the pipeline builder is referenced as a component not a UI label.

Reply:
All instances of operator must be Operator. Refer to the guideline: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#operator-name-capitalization

Reply:
> Arguably, places like this should be un-capitalized - "using the pipeline builder". However, it is not consistent in other places in this same doc set (I could find both "the operator" and "the Operator"), so your call whether to change it. If you do, change in all places where the pipeline builder is referenced as a component not a UI label.

Hi @mramendi , this is the confusion i had in the first place. However, upon going through it with Shipra, "Pipeline builder" is referring to there UI label in this case, unless otherwise where I may had have overlooked. Additionally, the capitalization is based on the appearance in the UI. kindly advise

Reply:
@masauso-lungu Has this issue been resolved?